homeassistant/components/binary_sensor/sleepiq.py

```diff
@@ -0,0 +1,15 @@
+"""
```

Comment:
Not sure the point of this file - know it's WIP but need more info to even understand